Portal/src/Datahub.Infrastructure/Services/Storage/AzureCloudStorageManager.cs

```diff
@@ -7,6 +7,7 @@
 using Datahub.Core.Storage;
 using Datahub.Infrastructure.Services.Security;
 using Datahub.Portal.Pages.Workspace.Storage.ResourcePages;
+using Microsoft.VisualStudio.Services.Common;
```

Comment:
We prob don't need this one

Reply:
IEnumerable.ForEach comes from there. Select doesn't work because addFile returns void; I could use a regular foreach loop there (line 243) but I think it's a bit cleaner this way.